### ISSUE TITLE
"No Items Found" EXTREMELY ANNOYING

### ISSUE THREAD

**Issue by turtle328** [PoE-TradeMacro/POE-TradeMacro#260]
Title says it all, I have no idea what I am doing wrong... What could possibly be causing this darn issue?!

**Comment by turtle328** [comment]
I FIXED IT! Just add http://poe.trade to trusted sites in Internet Settings and it worked like a charm. Would anyone care to explain why this happened now, and not before?

**Comment by turtle328** [comment]
I digged a little deeper, I think the problem is I am not getting a cf_clearance...

**Comment by Eruyome** [comment]
You don't need cookies. Are you using ersion 2.0.1?

**Comment by JBalanzar** [comment]
Unfortunately, I'm having a similar issue on the latest version (2.0.2). Is there any information we could provide to help find what's causing this?

Things I have tried:
Deleting the entire PoE-TradeMacro folder and re-downloading it.
Deleting the user folder in Documents.

**Comment by turtle328** [comment]
I had 2.0.1 when I had the issue, I'll try 2.0.2 but since JBalanzar still had the issue... I'm not optimisitc.

**Comment by Eruyome** [comment]
First of, tell me if ctrl + q on an item opens the correct search on poe.trade.

**Comment by JBalanzar** [comment]
It does, that is what I have been using for now.

**Comment by Eruyome** [comment]
This is copied from the forum and the line may be a bit different, basically add the code before the `Return, html` in function `TradeFunc_DoPostRequest`. Don't use the exe file.

> 1. Edit resources\ahk\TradeMacro.ahk and add at line 1141:
> FileDelete, %A_Scriptdir%\result.txt
> FileAppend, %html%, %A_Scriptdir%\result.txt
> 2. After a failed search (no items found) upload the file result.txt in the script root directory somewhere without making any kind of change to it and give me the link to this file.
> 
> Make sure to upload it only after a failed search.

**Comment by JBalanzar** [comment]
So I added that line of code to TradeFunc_DoPostRequest, but it doesn't seem to create a result.txt file after a failed search. The following is how my TradeFunc_DoPostRequest looks like: http://pastebin.com/cGG3nDq1

**Comment by Eruyome** [comment]
Did you run `Run_TradeMacro.ahk` after editing the file? And did you make sure not to edit `_TradeMacroMain.ahk`.

**Comment by JBalanzar** [comment]
I am running the `Run_TradeMacro.ahk` after editing the file and did not make the changes to `_TradeMacroMain.ahk.` 

The instructions you posted previously said to edit `resources\ahk\TradeMacro.ahk`. I searched for `TradeFunc_DoPostRequest` in the `resources\ahk\TradeMacroInit.ahk` and it did not find any functions with that name.

**Comment by Eruyome** [comment]
Ah sry, that was an error on my part, actually told you the wrong file :p 
Will take a look at the results.

**Comment by JBalanzar** [comment]
No worries, happens to the best of us. :)
Thanks! Correct, running 2.0.5 now and still no items found.

**Comment by Eruyome** [comment]
I opened your downloaded html as .html file in firefox, inspected it and compared it to a similiar search on poe.trade.

The resulting html is completely different.

![temp](https://cloud.githubusercontent.com/assets/16058242/23681492/71778c28-038f-11e7-835e-90575b05bf74.png)

**Comment by JBalanzar** [comment]
Crap, well I figured out why I wasn't getting results on my end. I am tethered through my phone (T-Mobile) using PDANet and have the Hide Tethering option enabled. After disabling it and enabling the native Tethering from the Android OS, I started getting results. So the method that PDANet uses to hide the tether usage is what's causing it not to return results. 

However, I am not sure if Turtle328's issue could be the same as mine.
